internal/controller/integration_test.go

```diff
@@ -966,20 +967,45 @@ var _ = Describe("Integration Test", func() {
 					WLMID:        "Test WLMID",
 				},
 			}
+
+			dmm = &nnfv1alpha1.NnfDataMovementManager{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      nnfv1alpha1.DataMovementManagerName,
+					Namespace: nnfv1alpha1.DataMovementNamespace,
+				},
+				Spec: nnfv1alpha1.NnfDataMovementManagerSpec{
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							Containers: []corev1.Container{{
+								Name:  "name-manager dummy",
```

Comment:
That doesn't look like a valid value for a k8s resource name.

Reply:
This is never actually used, but I fixed it.

Reply:
Just to clarify - this container spec is never used because the DMM reconciler doesn't actually run, but we do need to specify certain fields (name, image) to make the podspec happy.